src/import/FSHImporter.ts

```diff
@@ -178,10 +178,11 @@ export class FSHImporter extends FSHVisitor {
       // Collect the aliases and store in global map
       ctx.entity().forEach(e => {
         if (e.alias()) {
-          const [name, value] = e
-            .alias()
-            .SEQUENCE()
-            .map(s => s.getText());
+          const name = e.alias().SEQUENCE()[0]?.getText();
```

Comment:
As a minor point, I don't think the `?` is necessary, since `SEQUENCE()[0]` is the left side of the equals sign, which is always present.

Reply:
No longer an issue in 744155d, since this is handled with a lexical mode. Let me know what you think of the new implementation. One thing that would've been nice is if I could just redefine `SEQUENCE` in the relevant `mode ALIAS` block, but apparently that is not possible. Otherwise I think the approach you suggested @mint-thompson does do a better job avoiding any potentially confusing hacks.

Reply:
Became an issue again in https://github.com/FHIR/sushi/pull/880/commits/c2b7e1413b6cfc3b0096eb0a61dfdfc6f670e421, but fixed.